code/modules/surgery/organs/internal/cyberimp/augments_chest.dm

```diff
@@ -305,7 +311,8 @@
 	if(istype(tool, /obj/item/assembly/signaler/anomaly/grav))
 		user.balloon_alert(user, "core installed.")
 		athletics_boost_multiplier = 0.25
-		added_throw_range += 2
+		added_throw_speed += 4
```

Comment:
have you tested this? +6 throw speed to any item seems a bit insane

Reply:
I tested it out. The effect that anything that could embed, will embed, and do a pretty decent amount of damage at that. It lets you embed glass shards for one.

For an anomaly core item, it didn't seem TOO extreme. The actual embedding weapons that we have don't meaningfully become too much more or less lethal using it from my testing.